recipes/function2/all/conanfile.py

```diff
@@ -0,0 +1,33 @@
+import os
+
+from conans import ConanFile, tools
+
+
+class Function2Conan(ConanFile):
+    name = "function2"
+    description = "Improved and configurable drop-in replacement to std::function that supports move only types, multiple overloads and more"
+    topics = "function", "header-only"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Naios/continuable"
+    license = "Boost Software License 1.0"
+    settings = "os", "compiler", "arch", "build_type"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        tools.check_min_cppstd(self, "14")
```

Comment:
```suggestion
        if self.settings.compiler.cppstd:
            tools.check_min_cppstd(self, 14)
```

Reply:
i have a question about it there are several options how it is done in center:
```python
if self.settings.get_safe("compiler.cppstd"):
    tools.check_min_cppstd(self, 14)
```
```python
if self.settings.compiler.get_safe("cppstd"):
    tools.check_min_cppstd(self, 14)
```
```python
if self.settings.compiler.cppstd:
    tools.check_min_cppstd(self, 14)
```
```python
tools.check_min_cppstd(self, 14)
```

Even if using a number or a string is different. What should now be used and why?

Reply:
It does not matter, inside the function it is *stringized*.